ocaml/tests/common/test_common.ml

```diff
@@ -354,11 +354,12 @@ let make_sm ~__context ?(ref = Ref.make ()) ?(uuid = make_uuid ())
     ?(copyright = "") ?(version = "") ?(required_api_version = "")
     ?(capabilities = []) ?(features = default_sm_features)
     ?(host_pending_features = []) ?(configuration = []) ?(other_config = [])
-    ?(driver_filename = "/dev/null") ?(required_cluster_stack = []) () =
+    ?(driver_filename = "/dev/null") ?(required_cluster_stack = [])
+    ?(supported_image_formats = []) () =
```

Comment:
Is this a sensible default? Does this not force to set this parameter? Or is the empty list a special value?